tests/gcp/installer/util/test_data/expected_merged_forseti_conf_server_from_1.1.10.yaml

```diff
@@ -0,0 +1,206 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an input file that is used by the Forseti prerequisites setup
+# wizard to generate an actual forseti_conf.yaml. It's better to work off
+# the forseti_conf.yaml.sample than this one if you are editing the config
+# by hand.
+global:
+
+    # Remove or comment out the email properties if you do not want Forseti to
+    # send email notifications.
+    email_recipient: rec@forseti_test_gmail.com
+    email_sender: send@forseti_test_gmail.com
+    sendgrid_api_key: TESTING_API_KEY
+
+##############################################################################
+
+inventory:
+
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: ROOT_ID
```

Comment:
Hmmm, `ROOT_ID` looks like a literal?  Will the server handle this correctly?

Reply:
A root_resource_id is always a string literal e.g. "organizations/12345677890". The ROOT_ID here is only for testing purpose, the actual value will be generated during the installation process. I will update this to "organizations/12345677890" to be more clear.